examples/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "react-component-template-example",
+  "description": "Component template for React - usage example",
+  "version": "0.1.0",
+  "author": {
+    "name": "Sandstream Development",
+    "url": "https://github.com/sandstreamdev/react-swipeable-list/graphs/contributors"
```

Comment:
Please use {repository-name} pattern string everywhere so it can be replaced using simple find and replace.